apps/web/app/landing-page/Hero.tsx

```diff
@@ -38,16 +40,15 @@ export const Hero = () => {
 
   const { toast } = useToast();
 
-  useEffect(() => {
-    (async () => {
-      const response = await fetch(ApiRoutes.Subscribers)
-      if (response.ok) {
-        const count = await response.json()
-        setSubscribersCount(count)
-      }
-    })()
-  }, [])
+  const getSubscribersCount = async () => {
+    const response = await fetch(ApiRoutes.Subscribers)
+    if (response.ok) {
+      const count = await response.json()
+      return count
+    }
+  }
 
+  const { data:subscribersCount } = useQuery<number>( 'subscribres', async () => await getSubscribersCount())
```

Comment:
The lint seems broken, there shouldn't be a space before 'subscribers'.

It has also a typo.

@anti-duhring haven't you merged the previous PR into this one?

I kinda preferred the other name. (with count)